### PR TITLE
Include minSdkVersion for all plugins

### PR DESCRIPTION
**Pull request by lidongze91**
## Description

Add necessary AndroidManifest to support flutter sdk version and add more documentation regarding the minSdkVersion

### CODE REVIEW

File: packages/webview_flutter/README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ The plugin relies on Flutter's new mechanism for embedding Android and iOS views
 As that mechanism is currently in a developers preview, this plugin should also be
 considered a developers preview.
 
+**_Warning:_**
+evaluateJavaScript() function only supports SDK of 19 or greater for evaluating javascript.
```
Comment (collinjackson) [inline packages/webview_flutter/README.md]:
```suggestion
evaluateJavaScript() function only supports SDK of 19 or greater for evaluating JavaScript.
```

**Comment by lidongze91** [comment]
> Can you explain what the effect of this change is for developers? We'll need to update the pubspec.yaml and CHANGELOG entries for each of the affected plugins for release.

Right now flutter supports a minimum SDK version of 16 and all plugins should be able to match it, anything lower than that will probably cause some unwanted issues. We need to add those into Android Manifest and properly document it.

File: packages/webview_flutter/android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,2 +1,4 @@
-<manifest package="io.flutter.plugins.webviewflutter">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+          package="io.flutter.plugins.webviewflutter">
+  <uses-sdk android:minSdkVersion="16" />
```
Comment (amirh) [inline packages/webview_flutter/android/src/main/AndroidManifest.xml]: This should be 20

Comment (collinjackson) [reply packages/webview_flutter/android/src/main/AndroidManifest.xml]: Just for webview, right? Are there any other plugins that should be higher than 16?

Comment (amirh) [reply packages/webview_flutter/android/src/main/AndroidManifest.xml]: Sorry maps as well should be 20 though it's not included in this PR. These are the ones I'm aware of.

**Comment by lidongze91** [comment]
Hi Collin&Amir, could you guys do another review please? Not sure about the failing checks. It doesn't seem to be relevant for this PR.

File: packages/webview_flutter/README.md [hunk 3]
```diff
@@ -12,6 +12,9 @@ The plugin relies on Flutter's new mechanism for embedding Android and iOS views
 As that mechanism is currently in a developers preview, this plugin should also be
 considered a developers preview.
 
+**_Warning:_**
+evaluateJavaScript() function only supports SDK of 19 or greater for evaluating JavaScript.
```
Comment (mehmetf) [inline packages/webview_flutter/README.md]: If the minSDK of the plugin is 20, then this comment is not needed.

File: packages/android_intent/android/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -1,3 +1,4 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
   package="io.flutter.plugins.androidintent">
+  <uses-sdk android:minSdkVersion="16" />
```
Comment (mehmetf) [inline packages/android_intent/android/src/main/AndroidManifest.xml]: Could you please update the CHANGELOG.md of each plugin and increment their versions?

**Comment by collinjackson** [comment]
Looks good, I'm happy to merge this. I'd appreciate it if you could update the CHANGELOGs/pubspecs as Mehmet mentioned.

**Comment by ditman** [comment]
Thanks for the contribution! I'm closing this PR because the code has moved quite a bit since this was posted originally, and there's major conflicts right now (and I'm sure this ended up happening gradually to most plugins). I'm closing this to clean up our PR backlog. Thanks!

**Pull request closed without merging**